mvpa2/measures/rsa.py

```diff
@@ -199,12 +202,15 @@ class PDistTargetSimilarity(Measure):
           If True, return only the correlation coefficient (rho), otherwise
           return rho and probability, p.""")
 
-    def __init__(self, target_dsm, **kwargs):
+    def __init__(self, target_dsm, control_dsms = None, **kwargs):
```

Comment:
benefit is that it then automagically be listed by `__str__` and `__repr__` of this method